react/__mocks__/vtex.render-runtime.js

```diff
@@ -16,6 +16,11 @@ export const setMobileState = isMobile => {
   config.isMobile = isMobile
 }
 
+export const useRuntime = () => {
+  const [hints, setHints] = useState({ mobile: false, desktop: true })
```

Comment:
`useState` here is unnecessary beacuse youŕe just exporting the object hints...

Reply:
since this is a mock of the runtime, I think useState makes sense

Reply:
Nope, actually. I'll break it down for you. What was beeing done here:
1. After line 20, your hints will have a initial value that you passed to useState, that is `{ mobile: false, desktop: true }`.
2. You do nothing with this value and return it.

Sooo, you will export your initial value EVERY TIME. And you cannot change it, since you do not return the `setHints` as well.
Finally, this code can be refactored to: 
`export const useRuntime = ( ) => { hints: { mobile: false, desktop: true} } `

Got it ?